Lib/dis.py

```diff
@@ -495,7 +495,7 @@ def _get_instructions_bytes(code, varname_from_oparg=None,
                 argrepr = "to " + repr(argval)
             elif deop in hasjrel:
                 signed_arg = -arg if _is_backward_jump(deop) else arg
-                argval = offset + 2 + signed_arg*2
```

Comment:
Shouldn't this have been `offset + (signed_arg + 1) * 2` so that now it will be `offset + (signed_arg + 2) * 2`?